defaults/main.yml

```diff
@@ -81,6 +81,8 @@ beegfs_mgmt_host:  # Example: "{{ groups['cluster_beegfs_mgmt'] | first }}"
 beegfs_meta_path: "/data/beegfs/beegfs_meta"
 beegfs_meta_dev:  # /dev/sdb
 beegfs_meta_fstype:  # Example: "ext4"
+beegfs_meta_filesystem_opts: "-i 2048 -I 512 -J size=400 -O dir_index,filetype"
```

Comment:
what about fsopts

Reply:
Not sure I understnad. Do you mean rename the variable ?